std/didexchange/invitation/translate.go

```diff
@@ -21,6 +23,16 @@ func decodeB64(str string) ([]byte, error) {
 	return data, err
 }
 
+func didKeysToB58(keys []string) []string {
+	for index, key := range keys {
+		assert.That(strings.HasPrefix(key, "did:key"), true)
```

Comment:
I suppose the `true` argument is not needed, and I don't know what happens if the assertion is not satisfied because the code thinks arguments are values for format string 🤔 and there is no format string

Reply:
Wow, good catch 👍